Features/GoogleAnalytics/script.js

```diff
@@ -0,0 +1,35 @@
+/* global wpData, Cookies */
+
+import $ from 'jquery'
+import 'file-loader?name=vendor/js-cookie.js!js-cookie/src/js.cookie'
+
+const $gaOptoutLinks = $('.globalAction-optoutGa')
+const data = wpData
+
+const alreadyOptedOut = getOptoutCookie()
+
+if (alreadyOptedOut) {
+  $gaOptoutLinks.remove()
+  window['ga-disable-' + data.gaId] = true
+} else {
+  $gaOptoutLinks.on('click', function (e) {
+    e.preventDefault()
+
+    const confirmOptout = window.confirm(data.confirm)
+
+    if (confirmOptout) {
+      window['ga-disable-' + data.gaId] = true
+      window.alert(data.success)
+      setOptoutCookie()
+    }
+  })
+}
```

Comment:
I really don't like having two separate scripts (this file and the script.twig file) loaded in the same feature for no apparent reason. Either this functionality should be moved or the existing script should be extracted into this file. Defining and passing the same variables twice using two different approaches makes little sense.

Also it needs to be noted that the order in which these two scripts load is of importance. In this case it works fine, because this file is loaded first. But if that changed for some reason, it's not obvious that the opt out would not work anymore (since the `ga()` call would be triggered before the window variable is set).